CHANGELOG.md

```diff
@@ -1,6 +1,10 @@
+## 2.5.0-rc.2 / 2018-10-31
+
+* [BUGFIX] Fix accidental grouping change on targets page in web UI. #4806
```

Comment:
I'd say it is more a change than a bug fix: targets are now grouped by scrape config instead of job name and the job label is now displayed with all the other labels.

Reply:
Exactly my thoughts. It's definitely a CHANGE. Originally an accidental one but now fully loved and embraced…